### ISSUE TITLE
[Feature Request] Add pins/waypoints

### ISSUE THREAD

**Issue by Qwerty-Space** [toolbox4minecraft/amidst#475]
It would be nice to be able to add named pins on the map, and maybe be able to calculate the distance between them, so that we don't keep losing co-ordinates.

**Comment by Xavom** [comment]
Yes, I agrees adding pins would be so nice, and rectangles for highlighting things such as world border, reserved area, community area, specific biomes, etc. Accept a point coordinate, note, and symbol or rectangle width and height along with a selection to specify the point is to be the center point or a NE, NE, SE, SW corner. But if it is NW and SE corners that would fine. Text color and line color would be awesome too. 
Attached an example of what we do to share seed ideas from season to season. 
A few simple symbols would help as where I highlighted the strongholds. 
![Screenshot from 2019-07-21 20-26-02](https://user-images.githubusercontent.com/17793670/61600378-ed6a1380-abf5-11e9-8ace-6739a171412b.png)
